src/server/auth/server/auth_test.go

```diff
@@ -2570,6 +2566,56 @@ func TestOneTimePasswordExpires(t *testing.T) {
 	require.Nil(t, authResp)
 }
 
+func TestS3GatewayAuthRequests(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	fmt.Println("AAA")
```

Comment:
Leftover printfs in this function should be removed before merge.

Reply:
Right. I'm still trying to debug an issue on this PR.